Buildings/Resources/Scripts/Dymola/Applications/DHC/EnergyTransferStations/Examples/CoolingIndirectClosedBuildingLoop.mos

```diff
@@ -0,0 +1,62 @@
+﻿// Simulate
+simulateModel("Buildings.Applications.DHC.EnergyTransferStations.Examples.CoolingIndirectClosedBuildingLoop", tolerance=0.001, stopTime=86400, numberOfIntervals=2000, resultFile="CoolingIndirectClosedBuildingLoop");
```

Comment:
need to change the tolerance to 1e-6. As suggested from the CI test:
```
ValueError: Found mos file=./Resources/Scripts/Dymola/Applications/DHC/EnergyTransferStations/Examples/CoolingIndirectClosedBuildingLoop.mos without tolerance defined.A minimum tolerance of 1e-6 is required for JModelica.
```

Reply:
@JayHuLBL the tighten tolerance causes a significant amount of noise in the simulation. I'm working to see if/how I can tune the pump/controller parameters to reduce/eliminate this noise. If this proves to be problematic however, how critical is this tolerance reduction? Any suggestions you may have as well to address this would be much appreciated.

Reply:
By setting the `dp_nominal` of the pump could partially mitigate the issue. However, more tuning may need to done to eventually solve it.